frontend/src/app/analytics/Analytics.tsx

```diff
@@ -199,82 +184,99 @@ export const Analytics = (props: Props) => {
               {dateRange === "custom" && (
                 <div className="grid-row grid-gap margin-top-2">
                   <div className="grid-col-4">
-                    <DatePicker
-                      name="startDate"
-                      label="Begin"
-                      onChange={(date?: string) => {
-                        if (date && date.length === 10) {
-                          const newDate = new Date(date);
-                          setStartDate(
-                            setStartTimeForDateRange(
-                              newDate
-                            ).toLocaleDateString()
-                          );
+                    <label className={classNames("usa-label")}>Begin</label>
+                    <input
+                      id={"startDate"}
+                      data-testid={"startDate"}
+                      type={"date"}
+                      max={formatDate(new Date())}
+                      className={classNames("usa-input")}
+                      aria-label={"Enter start date"}
+                      onChange={(e) => {
+                        if (Date.parse(e.target.value)) {
+                          const d = moment(e.target.value).toDate();
+                          const startDateString = setStartTimeForDateRange(
+                            new Date(d)
+                          ).toLocaleDateString();
+                          setStartDate(startDateString);
                         }
                       }}
-                      noHint
-                    />
+                      defaultValue={formatDate(new Date(startDate))}
+                    />{" "}
                   </div>
                   <div className="grid-col-4">
-                    <DatePicker
-                      name="endDate"
-                      label="End"
-                      onChange={(date?: string) => {
-                        if (date && date.length === 10) {
-                          const newDate = new Date(date);
-                          setEndDate(
-                            setEndTimeForDateRange(newDate).toLocaleDateString()
-                          );
+                    <label className={classNames("usa-label")}>End</label>
+                    <input
+                      id={"endDate"}
+                      data-testid={"endDate"}
+                      type={"date"}
+                      min={formatDate(new Date(startDate))}
```

Comment:
FYI: from a typing in the input perspective this min/max doesn't seem to do anything but it does affect the date picker that shows up
<img width="781" alt="Screen Shot 2022-09-21 at 11 18 14 AM" src="https://user-images.githubusercontent.com/20211771/191557851-f0c9b58c-9ac4-413c-a167-e2c4ebd0828d.png">

Reply:
Right now there is not validation on the fields (there wasn't before either), and so there really isn't any easy way to prevent a user typing a date outside of the range. The max/min is mostly for that dropdown date picker.